### PR TITLE
docs(v2): add documentation for multiple blogs

### LINKED ISSUE #2481: docs(v2): document about allowing multiple blogs
## 🚀 Feature

I want to use docusaurus to organize my blog posts, which have different categories.

If I put these blog posts under the docs folder, I will need to manually add these blog posts to the sidebar, that would be a burdensome task.

If there is only one blog folder, I will not be able to sort these blog posts well.

## Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?
Yes.

### PR DESCRIPTION
**Pull request by teikjun**
## Motivation

Add documentation about how to set up multiple blogs. 

This PR closes #2481. It also fixes a minor typo. 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

The changes can be viewed on Netlify here: 
https://deploy-preview-2933--docusaurus-2.netlify.app/docs/blog/#multiple-blogs

## Related Information

As noted in #2883, the user needs to add `require.resolve(<plugin-name>)` to the plugin name, in addition to the instructions stated in the documentation.
But I have excluded this to maintain consistency with the [plugins documentation](https://v2.docusaurus.io/docs/using-plugins/).

Should I make another PR to add in `require.resolve` for all plugin names?

### CODE REVIEW

File: website/docs/blog.md [hunk 1]
```diff
@@ -163,6 +163,25 @@ Make sure there's no `index.js` page in `src/pages` or else there will be two fi
 
 :::
 
+### Multiple blogs
+
+You can add another blog by specifying your own blog plugin in the `plugins` option. Set the `routeBasePath` to the URL route that you want your blog to be accessed on. Also, set `path` to the path to your blog folder.
```
Comment (yangshun) [inline website/docs/blog.md:168]: Recommend using this:

 By default, the classic theme assumes only one blog per website and hence includes only one instance of the blog plugin. If you would like to have multiple blogs on a single website, it's possible too! You can add another blog by specifying another blog plugin in the `plugins` option for `docusaurus.config.js`.  Set the `routeBasePath` to the URL route that you want your second blog to be accessed on. Note that the `routeBasePath` here has to be different from the first blog or else there could be a collision of paths! Also, set `path` to the path to the directory containing your other second blog's entries.

File: website/docs/blog.md [hunk 2]
```diff
@@ -163,6 +163,25 @@ Make sure there's no `index.js` page in `src/pages` or else there will be two fi
...
+
+```js {4-10} title="docusaurus.config.js"
+module.exports = {
+  // ...
+  plugins: [
+    [
+      '@docusaurus/plugin-content-blog',
+      {
+        routeBasePath: 'my-blog', // URL route for your blog
```
Comment (yangshun) [inline website/docs/blog.md:177]: 'my-second-blog'

File: website/docs/blog.md [hunk 3]
```diff
@@ -163,6 +163,25 @@ Make sure there's no `index.js` page in `src/pages` or else there will be two fi
...
+        path: './my-blog', // path to your blog folder
```
Comment (yangshun) [inline website/docs/blog.md:178]: `'./my-second-blog'`

File: website/docs/blog.md (hunk 2 above)
Comment (yangshun) [inline website/docs/blog.md:177]: Please use comments similar to https://v2.docusaurus.io/docs/using-plugins#docusaurusplugin-content-blog to better elaborate on the options.

**Comment by teikjun** [comment]
I have included your suggested changes, it looks much better now! :D

**Pull request merged**